rebar.config

```diff
@@ -1,3 +1,8 @@
+{deps, [
+    {goldrush, ".*",
+        {git, "https://git-wip-us.apache.org/repos/asf/couchdb-goldrush.git",
+        {branch, "master"}}}
```

Comment:
Technically, we can, but we'll have to do more work on release in order to update all the rebar.config for all the deps. Hopefully, this will be need to do rarely and easy to detect. 

For current state of things, we already have deps defined in subproject [[1]](https://github.com/apache/couchdb-folsom/blob/master/rebar.config#L3-L6) [[2]](https://github.com/apache/couchdb-couch-log/blob/master/rebar.config#L1-L3) and this works fine.